Message Orchestration/Canvas Sync Alt Text (#3293)
braze-inc/braze-docs: _docs/_partners/canvas_steps/facebook_audience_sync.md

In the Braze dashboard, go to __Technology Partners__ and select __Facebook__. In the Facebook Audience Export module, click __Connect Facebook__.

> ![An image of the Facebook technology page in Braze.][4]{: style="max-width:70%;"}

Reviewer: ```suggestion
![Facebook technology page in Braze that includes an Overview module and Facebook Audience Export module with the Connected Facebook button.][4]{: style="max-width:70%;"}
```

Add a step in your Canvas, select the dropdown at the top of the step, and select __Facebook Audience Step__.

> ![An image of what the steps written above look like in the Canvas UI.][11]{:style="max-width:70%"}

Reviewer: ```suggestion
![Workflow of the previous steps to add a Facebook Audience step in Canvas.][11]{:style="max-width:70%"}
```

__Create a New Audience__<br>
Enter a name for the new custom audience, select __Add Users to Audience__ and select which fields you would like to sync with Facebook. Next, save your audience by clicking the __Create Audience__ button at the bottom of the step editor.

> ![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account is selected and a new audience is created.]({% image_buster /assets/img/fb_audience_sync/create_audience.png %})

Reviewer: ```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account is selected and a new audience is created.]({% image_buster /assets/img/fb_audience_sync/create_audience.png %})
```

Users will be notified at the top of the step editor if the audience is created successfully or if errors arise during this process. Users can also reference this audience for user removal later in the Canvas journey because the audience was created in draft mode. 

> ![An image of the alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/fb_audience_sync/new_audience.png %})

Reviewer: ```suggestion
![An alert that appears once a new audience is created in the Canvas step.]({% image_buster /assets/img/fb_audience_sync/new_audience.png %})
```

__Sync with an Existing Audience__<br>
Braze also offers the ability to either add or remove users from existing Facebook custom audiences to ensure that these audiences are up-to-date. To sync with an existing audience, type the existing audience name in the dropdown and choose whether you want to __Add to the Audience__ or __Remove from the Audience__. Braze will then either add or remove users in near real-time as they enter the Facebook Audience Step. 

> ![An image of the Custom Audience Canvas step expanded. Here, the desired Ad account and existing audience is selected.]({% image_buster /assets/img/fb_audience_sync/add_audience.png %})

Reviewer: ```suggestion
![Expanded view of the Custom Audience Canvas step. Here, the desired Ad account and existing audience are selected.]({% image_buster /assets/img/fb_audience_sync/add_audience.png %})
```

The __History__ tab of the custom audience in the Facebook Audience Manager will reflect the number of users sent to the audience from Braze. If a user re-enters the step, they will be sent to Facebook again.

> ![An image of what the audience deatils and history looks like for a given Facebook audience.][9]{: style="max-width:80%;"}

Reviewer: ```suggestion
![Audience details and the History tab for a given Facebook audience that includes an Audience History table with columns for the activity, activity details, items changed, and the date and time.][9]{: style="max-width:80%;"}
```
